src/apprt/gtk/Window.zig

```diff
@@ -559,6 +559,17 @@ pub fn toggleWindowDecorations(self: *Window) void {
     self.headerbar.setVisible(new_decorated);
 }
 
+/// Toggle the background opacity for this window.
+pub fn toggleBackgroundOpacity(self: *Window) void {
+    if (self.app.config.@"background-opacity" >= 1) return;
+
+    if (c.gtk_widget_has_css_class(@ptrCast(self.window), "background") == 1) {
+        c.gtk_widget_remove_css_class(@ptrCast(self.window), "background");
+    } else {
+        c.gtk_widget_add_css_class(@ptrCast(self.window), "background");
+    }
+}
+
```

Comment:
This doesn't actually update the config for the OpenGL renderer, so the actual terminal screen will still render as if they're transparent (and likely have wrong colors)

Reply:
Good point, didn't even think about that. Let me take a look at that as well